brand/admin.py

```diff
@@ -335,3 +386,15 @@ def get_form(self, request, obj=None, *args, **kwargs):
             "user_prompt": "write your question here",
         }
         return form
+
+
+@admin.register(Contact)
+class ContactAdmin(admin.ModelAdmin):
```

Comment:
I would personally have added a clickable link to the brand, since that will make navigating easier. You can find out how it's done on line 247:

`@admin.display(description="related datasources")`

Reply:
I _think_ that will allow you to remove the `nested_admin` requirement too. The fewer dependencies the better

Reply:
In this case, I can see that you're following the acceptance criteria to the letter. When it comes to util type stuff like this that the user won't see and especially when the ticket hasn't been extremely carefully groomed, this is an opportunity to push back on the criteria a bit. Sometimes the necessary code is different from what everyone expects.